src/components/CharacterButton.vue

```diff
@@ -234,6 +232,11 @@ export default defineComponent({
       return character;
     });
 
+    const isSelectedItem = (characterInfo: CharacterInfo) =>
+      selectedCharacter.value != undefined &&
+      characterInfo.metas.speakerUuid ===
+        selectedCharacter.value?.metas.speakerUuid;
+
```

Comment:
あ、ここはVue独特な記法で、`computed`でラップする必要があったりします！
（computedでラップすることで`selectedCharacter`の値が変わった際に自動で再評価されます。）

たぶんこんな感じ･･･？
```suggestion
    const isSelectedItem = computed((characterInfo: CharacterInfo) =>
      selectedCharacter.value != undefined &&
      characterInfo.metas.speakerUuid ===
        selectedCharacter.value?.metas.speakerUuid;
    )

```

Reply:
Vueに詳しくないのでよくわかっていないのですが、
```vue
const isSelectedItem = computed(
  (charcterInfo: CharacterInfo) =>
    selectedCharacter.value != undefined &&
    charcterInfo.metas.speakerUuid ===
      selectedCharacter.value?.metas.speakerUuid
);
```
👆だと関数が返せてないので、
```vue
const isSelectedItem = computed(
  () => (charcterInfo: CharacterInfo) =>
    selectedCharacter.value != undefined &&
    charcterInfo.metas.speakerUuid ===
      selectedCharacter.value?.metas.speakerUuid
);
```
こういう感じにしてみようと思います。
ただこれで正しくキャッシュされているかどうかわかってないです。~調べた限りだとキャッシュされてない~

Reply:
computedは[算出プロパティ](https://ja.vuejs.org/guide/essentials/computed.html)ですね。
多分元のままであってると思います。

Reply:
レビューありがとうございます!

sevenc-nanashiさんの教えてくださったページを見ると、
computedで包んで上げても特に変化なさそうです。
もとのやつをpushし直しておきます。

正直Vueなんにもわかってないので、教えていただけて非常に助かります。

Reply:
あ！本当ですね、関数でした！！
実行回数なども見てみましたが同じだったので、普通に関数で良さそうに思いました！！